apisix/wasm.lua

```diff
@@ -26,8 +26,14 @@ local schema = {
             type = "string",
             minLength = 1,
         },
+        setting = {
+            type = "object",
+        },
+    },
+    oneOf = {
+        {required = {"conf"}},
+        {required = {"setting"}},
```

Comment:
Why not use the same option `conf`,  use `type` to check if conf is a object or string

Reply:
Makes sense, I'll modify it later.

Reply:
https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L289